README.md

```diff
@@ -83,7 +83,7 @@ Run the following script to download the documentation tarballs from the Daml an
 
 ### Live preview
 
-Run the `live-preview.sh` script to render a local view of the site. The html files are in `docs/workdir/build`. Edits to `docs/index/` files should be immediately reflected.
+Run the `live-preview.sh` script to render a local view of the site. The html files are in `docs/workdir/build`. Rerun the script to view any edits to `docs/index/` files.
```

Comment:
```suggestion
Run the `live-preview.sh` script to render a local view of the site. The html files are in `docs/workdir/build`. Changes to files under `docs/index/` should reflect immediately, though you'll have to kill and rerun the script when adding or removing a file.
```

I'd rather fix the issue than document broken behaviour; please confirm that #65 fixes this for you.

Reply:
Hi, `live-preview` still requires a restart to pick up changes.

Reply:
Can you confirm you still see that bug on the `fix-live-preview` branch?

Reply:
Still have to rerun the script to see changes.

Reply:
I'm still not comfortable documenting broken behaviour.

Reply:
Until it's fixed, this is how it behaves.